home_work_2/addition_multiplication_fractions.py

```diff
@@ -0,0 +1,57 @@
+# Задача №2 домашнего задания 2. Сложение и умножение дробей.
+
+from fractions import Fraction
+
+
+def gcd(a, b):
+    if a == 0:
+        return b
+    return gcd(b % a, a)
+
+
+def fraction_reduction(num, den):
+    com_div = gcd(num, den)
+    num /= com_div
+    den /= com_div
+    return int(num), int(den)
+
+
+def str_split(my_str):
+    my_str = my_str.split('/')
+    a = int(my_str[0])
+    b = int(my_str[1])
+    return a, b
```

Comment:
Можно значительно проще. По сути, вообще функция `str_split` не нужна.
```suggestion
    return map(int, my_str.split('/'))
```